sources/packages/backend/libs/integrations/src/cra-integration/cra.integration.service.ts

```diff
@@ -149,7 +149,7 @@ export class CRAIntegrationService extends SFTPIntegrationBase<CRASFTPResponseFi
     const statusRecords: CRAResponseStatusRecord[] = [];
     const totalIncomeRecords: CRAResponseTotalIncomeRecord[] = [];
     fileLines.forEach((line: string, index: number) => {
-      const lineNumber = index + 1;
+      const lineNumber = index + 2; // Take into account the removed header line.
```

Comment:
maybe we  can move  the comment above the line 152

Reply:
It would be ok for me either way 😉